internal/controllers/terraformlayer/conditions_test.go

```diff
@@ -208,3 +209,140 @@ var _ = Describe("TerraformLayer", func() {
 	// 	})
 	// })
 })
+
+func TestExponentialRetry(t *testing.T) {
```

Comment:
Since your function is inside `controller.go` the test should be in a `controller_test.go`